src/services/notification.js

```diff
@@ -38,6 +43,20 @@ class Notification extends EventEmitter {
     if (type in this) return this[type](payload);
   }
 
+  /**
+   * Initialises the event
+   *
+   * @param {string} type - the type of notification
+   * @param {Function} notificationHandler - function that handles defined notification
+   *
+   * @return {Notification} -instance of notification
+   */
+  addNotification(type, notificationHandler) {
```

Comment:
Great Job. Please go through your Js Docs, I believe we are to have `@functions` stated based on our team convention